docs/introduction/quick-start.md

```diff
@@ -142,8 +145,6 @@ tendermint node --home ./mytestnet/node3 --proxy_app=kvstore --p2p.persistent_pe
 
 Note that after the third node is started, blocks will start to stream in
 because >2/3 of validators (defined in the `genesis.json`) have come online.
-Seeds can also be specified in the `config.toml`. See [this
-PR](https://github.com/tendermint/tendermint/pull/792) for more information
-about configuration options.
+Seeds can also be specified in the `config.toml`. See [here](../tendermint-core/configuration.md) for more information about configuration options.
```

Comment:
relative link

Reply:
relatvie links are in fact the way to go, see #2398